README.md

```diff
@@ -76,6 +80,21 @@ const buf = message.SomeMessage.encode({
 See the [Google Protocol Buffers docs](https://developers.google.com/protocol-buffers/) for more information about the
 available types etc.
 
+## Properties
+
+Decoded object properties can be interacted with using accessor methods:
+
+```javascript
+ const obj = messages.WithOptional.decode(messages.WithOptional.encode({}))
+
+obj.hasPayload() // false
+obj.getPayload() // ''
+obj.setPayload('hello world')
+obj.getPayload() // 'hello world'
+obj.clearPayload()
+obj.getPayload() // undefined
+```
+
 ## Performance
```

Comment:
Update the benchmarks with your new ones?

Reply:
Done!